### DISCUSSION TITLE
Single-Player Game Without Server

### DISCUSSION THREAD

**Discussion by i-jared** [jeffreylanters/react-unity-webgl#346] [Help Wanted!]
As I'm trying to release my 2d single player unity game using this package, I've read a lot about web servers. Is it possible to keep the game entirely locally contained on my website? Or do I need a separate server that it sends http requests to as I play? 

I'd like to release the game in such a way that it doesn't need to make any http requests. Is that possible?

**Comment by jeffreylanters** [comment]
Unless I understand you incorrectly. You're going to need any kind of server in order to achieve multiplayer. This package only provides an API to interact with React and the web browser.
